test/common/router/router_ratelimit_test.cc

```diff
@@ -173,7 +173,7 @@ TEST_F(RateLimitConfiguration, NoRateLimitPolicy) {
   }
   )EOF";
 
-  SetUpTest(json);
+  setUpTest(json);
```

Comment:
There's a weird lexical/grammar thing going on here with `setUp`; I would write it as `setupTest`. I think you are paying homage to gtest's `SetUp`, which is also pretty weird IMHO; is it a pun on "up"?

Reply:
:)  I was just changing one character in an already-existing test-method to reflect the fact that this is a locally defined class method, not associated with gtest at all (and easy to confuse with it). So it should use Envoy's convention of a lower-cased first character.

But you're right, this looks weird, so I might as well clean this up and change it to setupTest. Any other issues?

Reply:
Nup, that's all good. I'd like to make more of these warnings act as errors though, I always prefer cleanups that don't need to ever be done again :) Thanks for doing the grungey work in this PR.